### DISCUSSION TITLE
Systems Icons Hidden

### DISCUSSION THREAD

**Discussion by kjagarwal1** [martinet101/ElevenClock#1317] [Q&A]
Since I have a wide screen monitor, I like having the text in one row and relatively large. I formatted the clock to my liking, but now it's covering system icons. If I click on the arrow, the entire clock is turned off and it goes back to original windows clock. Is there any way to fix this?

As of now, I made the text longer (but even that has limitations; explained later) and used a setting to hide on hover. Not ideal, but at least the system doesn't mess up.

The second issue is that when I add all of the text to one line and make the font larger, half of it doesn't show up. It's clearly because if it's too long, the text just gets cut off where the overlay ends.

I understand this is not how the app was made, but this was possible with T-Clock, so just wanted to see what the options were with this app.

**Comment by marticliment** [comment]
No, ElevenClock cannot move System Icons to the left to have more space for the clock, due to the way in which ElevenClock works
